src/charts/c3/c3.directive.js

```diff
@@ -0,0 +1,52 @@
+/**
+ * @ngdoc directive
+ * @name patternfly.charts.directive:c3Chart
+ *
+ * @description
+ *   Directive for wrapping c3 library
+ *
+ *
+ * @param {expression} chartConfig the c3 configuration options for the chart
+ * @param {string} id the ID iof the container that the chart should bind to
+ *
+ * @example
+<file name="index.html">
+  <div id="chartId">
+    <div c3-chart id="chartId"  config="chartConfig"></div>
+  </div>
+ </file>
+
+<file name="script.js">
+ function ChartCtrl () {
+    var vm = this;
+    vm.chartId = 'myChartId';
+    vm.chartConfig = { 'labels': ['Created','Deleted'], 'tooltipSuffixes': ['Container Group','Container Group'] };
+ }
+ </file>
+ */
+(function(c3){
+  'use strict';
+
+  angular.module('patternfly.charts')
```

Comment:
> I wonder if we ought to put c3 into its own module? Maybe patternfly.charts.c3? That would allows us to list it as a dependency here.
> 
> Or we could create an angular-c3 or some such as it's own upstream project.

Scratch that.  Are we not able to use either of these for wrapping c3?

https://github.com/GraFiddle/angular-chart
https://github.com/jettro/c3-angular-directive